### DISCUSSION TITLE
`monitors`, `helpers, `image_capture`, and other libraries

### DISCUSSION THREAD

**Discussion by David-OConnor** [openvinotoolkit/openvino#20470] [Q&A] [answered]
Hi! I am attempting to get the OpenModelZoo programs running, using Python. They run when run from the OMZ folder, but not when attempting to adapt them to a custom code base: The following dependencies, listed in `face_recognition_demo.py`, are unable to be found:

  - monitors
  - helpers
  - images_capture
  - model_api

I can't find them either on pypi, or in the OMZ folder, nor can my IDE.  How should I proceed? Thank you!

**Comment by Wovchena** [comment, marked as answer]
Hi. They are here https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/common/python/openvino/model_zoo/model_api

**Comment by David-OConnor** [comment]
Thank you very much; solved!
